### PR TITLE
Newsweek website clone for milestone 2

### PR DESCRIPTION
**Pull request by mahbubul14**
This website is a clone of the original website
The main focus is to build 2 sections (Header and Footer) for three screens (Mobile, Tablet, and laptop)

# Framework used here:

* Bootstrap

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -73,8 +73,638 @@
...
+                <div class="culture d-md-flex justify-n">
+                  <div>
+                    <a href="#" class="text-decoration-none">
+                      <div class="img-box">
+                        <img class="l-img" src="./images/culture2.png" alt="" />
+                        <div class="bottom-left">
+                        <p class="bottom-left text-dark fw-bold bg-light-sky">Music</p>
+                        </div>
+                      </div>
+                        <p class="fw-bold text-secondary m-2">These Are the Top 100 Songs of 2020</p>
```
Comment (george-shammar) [inline index.html:337]: - [ ] Every title in this category should be an `h` tag. Please use an `h` as well for other titles in this section.

File: index.html [hunk 2]
```diff
@@ -73,8 +73,638 @@
...
+            </div>
+            <div class="p-2 mb-2" id="my-turn">
+              <section>
+                <h6 class="text-secondary">MY TURN</h6>
+                <div>
+                  <div class="myturn d-md-flex justify-n">
+                    <div>
+                      <a href="#" class="text-decoration-none">
+                        <img class="l-img" src="./images/myturn1.jpg" alt="" />
+                        <p class="fw-bold text-secondary">'I've Rescued Hundreds of Special Needs Dogs'</p>
```
Comment (george-shammar) [inline index.html:306]: - [ ]  Every title in this category should be an `h` tag. Please use an` h` tag as well for other titles in this section.

File: index.html [hunk 3]
```diff
@@ -73,8 +73,638 @@
...
+                </div>
+              </div>
+            </div>
+            <div class="p-2 mb-2" id="sponsored-insights-1">
+              <div>
+                <div>
+                  <div class="img-box">
+                      <a href="#"><img class="feat-img" src="./images/insights1.jpg" alt=""/></a>
+                    <div class="bottom-left nowrap">
+                      <p class="bottom-left text-primary fw-bold bg-grey p-2 ">SPONSORED INSIGHTS</p>
```
Comment (george-shammar) [inline index.html:210]: - [ ]  Every title in this category should be an` h `tag. Please use an `h` tag as well for other titles in this section.

**Review by george-shammar — CHANGES_REQUESTED** [review]
### **STATUS: CHANGES REQUIRED** :recycle: :recycle: 

**You've done a really awesome job in this project and your effort is commendable.** :ok_hand: 

**Great job.** :+1: 

**Now, let's make this milestone even better.** 

**Please see specific changes in the inline comments.**

- [ ] Please use `h` tag for all the titles in the sections I specified in the inline comments.
- [ ] In your tablet version, "The Debate" section is aligned in comparison to the original. Please see below:

Original
image

Yours
image

- [ ] In the mobile version, this should be the order of the sections:
image

Your order currently is :
Top Story => Featured Stories => Sponsored Insights => Top Story => My Turn => Culture $ Travel => More Stories => Latest News => The Debate => Opinion => Sponsored Insights => Free Sign Up => Sponsored Insights.

Please fix it.

**Please submit another CRR when you're ready.**

:red_circle: **As described in the** [Code reviews limit Policy ](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), **you have 2 code reviews left for this project.**

**Happy coding.** :computer: :computer_mouse:

**Review by sinansevgi — APPROVED** [review]
# Project status: Approved! :heavy_check_mark: 
----
Good job! Keep it up! You fixed all the issues raised by the previous TSE.

You can merge this pull request and continue with your journey :wave: 

----
Sinan Sevgi

**Pull request merged**